src/lib/settings/types/ClientSettings.ts

```diff
@@ -40,5 +40,7 @@ export namespace ClientSettings {
 
 		export const GnomeRestaurantCostBank = T<O.Readonly<ItemBank>>('gnome_res_cost');
 		export const GnomeRestaurantLootBank = T<O.Readonly<ItemBank>>('gnome_res_loot');
+
+		export const RoguesDenStaminas = T<O.Readonly<ItemBank>>('economyStats.roguesDenStaminas');
```

Comment:
follow the naming of the gnomerestaurant ones 

the input/cost/etc is called "cost" and we use snake_case now, and no `economyStats.` here. e.g. `rogues_den_cost`